src/runOnPush.js

```diff
@@ -5,16 +5,19 @@ import {execCmd} from './execCmd';
 import {ownerAndRepo, extraPermGithub, type Context} from './setup';
 import {PUSH, GERALD_COMMIT_COMMENT_HEADER} from './constants';
 
-const makeCommitComment = async (
-    peopleToFiles: {[string]: Array<string>, ...},
-    commitSHA: string,
-) => {
-    const names: string[] = Object.keys(peopleToFiles);
-    if (peopleToFiles && names.length) {
+type NameToLabelToFiles = {[name: string]: {[label: string]: string[], ...}, ...};
```

Comment:
Can we put this somewhere shared so as to not dupe?

Reply:
If you want.  I like having it in each place so it's easier to find when reading the file; flow won't let them get out of sync.  But I know some people feel very militant about DRY!